src/main/java/com/sda/zdjavapol107/szubienica/dataBaseInsert/DatabaseSloganInsert.java

```diff
@@ -0,0 +1,54 @@
+package com.sda.zdjavapol107.szubienica.dataBaseInsert;
+
+import com.sda.zdjavapol107.HibernateFactory;
+import com.sda.zdjavapol107.szubienica.repository.dao.HangmanSloganDao;
+import com.sda.zdjavapol107.szubienica.repository.model.HangmanSlogan;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+public class DatabaseSloganInsert {
```

Comment:
otwieracie tutaj sesje i transakcje, 
od takich rzeczy jest Dao ! :D i w wiekszosci ten kod powinien zostac przeniesiony do sloganDao pod jakas funkcje initialize czy cos :) 
ewentualnie initialize moze przyjac liste co ma zainicjalizowac (albo saveAll mozecie to nazwac jezeli przekazecie liste)